packages/auth/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
I wonder if as a monorepo, you should have a master linter that is shared with every package.

Reply:
I remember that at the beginning there was a discussion about this. Santiago decided to keep the linter in each package... I think to later make it a microservice, I don't remember why. But all the squads integrate their linter.